--- .vagrant.d/gems/gems/vagrant-openstack-plugin-0.11.1/lib/vagrant-openstack-plugin/action/create_server.rb	2015-03-03 19:46:30.166688299 +0000
+++ .vagrant.d/gems/gems/vagrant-openstack-plugin-0.11.1/lib/vagrant-openstack-plugin/action/create_server.rb	2015-03-03 19:42:16.615807383 +0000
@@ -20,7 +20,7 @@ module VagrantPlugins
           config   = env[:machine].provider_config
 
           # Find the flavor
-          env[:ui].info(I18n.t("vagrant_openstack.finding_flavor"))
+          env[:ui].info(I18n.t("vagrant_openstack.finding_flavor") + " #{config.flavor}")
           flavor = find_matching(env[:openstack_compute].flavors.all, config.flavor)
           raise Errors::NoMatchingFlavor if !flavor
 
@@ -68,6 +68,11 @@ module VagrantPlugins
                 # Match the OpenStack network to a corresponding
                 # config.vm.network option.  If there is one, use that for its
                 # IP address.
+                env[:ui].info("#{env[:machine].config.vm.networks[i][0]}")
+                if env[:machine].config.vm.networks[i][0] == :forwarded_port
+                  i=i+1
+                  env[:ui].info("#{env[:machine].config.vm.networks[i][0]}")
+                end
                 config_network = env[:machine].config.vm.networks[i]
                 if config_network
                   ip_address = config_network[1][:ip]
